solutions/ids/services.yaml

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #########
-# Cloud IDS API
+# AC-17(1), SC-7(9), SC-18(1), SI-3(2), SI-3(4), SI-3(7), SI-4(4), SI-4(5), SI-4(7) - Enable Cloud IDS APIs in host project that contains shared vpc network
```

Comment:
just thinking for the `Service` kinds, if we should update the project-id annotation to use projectRef instead...?

Reply:
included in latest commit